client/app/tests/components/documentForm.spec.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { expect } from 'chai';
+import chai from 'chai';
+import chaiEnzyme from 'chai-enzyme';
+import 'babel-polyfill';
+import sinon from 'sinon';
+import { mount, shallow } from 'enzyme';
+import { DocumentForm } from '../../src/components/documents/DocumentForm';
+
+chai.use(chaiEnzyme());
+
+describe('<DocumentForm />', () => {
+  const props = {
+  title: 'document',
+  content: 'the boy is going to school',
+  userId: 3,
+
+};
+  const wrapper = shallow(<DocumentForm {...props} />)
+  it('Should render a modal element for the document', () => {
+    expect(wrapper.find('Modal')).to.have.length(1);
+    expect(wrapper.find('Modal').children()).to.not.be.undefined;
+  });
+  it('Should have a input for taking in values as props', () => {
+    expect(wrapper.find('input')).to.not.be.undefined;
```

Comment:
Don't just check that the component exists, also check that it has the expected props passed to it.